src/lib/divider/divider.ts

```diff
@@ -1,29 +1,69 @@
-import { attachShadowTemplate, CustomElement } from '@tylertech/forge-core';
-import { BaseComponent, IBaseComponent } from '../core/base/base-component';
+import { attachShadowTemplate, coerceBoolean, CustomElement } from '@tylertech/forge-core';
+import { BaseComponent } from '../core/base/base-component';
 import { DIVIDER_CONSTANTS } from './divider-constants';
+import { IWithDefaultAria, WithDefaultAria } from '../core/mixins/internals/with-default-aria';
+import { IWithElementInternals, WithElementInternals } from '../core/mixins/internals/with-element-internals';
 
 import template from './divider.html';
 import styles from './divider.scss';
+import { setDefaultAria } from '../constants';
```

Comment:
Minor code style thing, but could we move this to the group of imports above? Just helps us to differentiate normal module imports from templates and styles in component files.